### ISSUE TITLE
'Uint8Array' is not defined

### ISSUE THREAD

**Issue by AlexWayfer** [eslint/eslint#11451]
**Tell us about your environment**

* **ESLint Version:** `v5.14.1`
* **Node Version:** `v11.9.0`
* **npm Version:** `6.5.0`

**What parser (default, Babel-ESLint, etc.) are you using?**

Default.

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```yaml
extends: 'eslint:recommended'
env:
  browser: true
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
var dataArray = new Uint8Array(bufferLength);
```

```bash
eslint *.js
```

**What did you expect to happen?**

No errors about `Uint8Array`.

**What actually happened? Please include the actual, raw output from ESLint.**

```
error    'Uint8Array' is not defined   no-undef
```

**Are you willing to submit a pull request to fix this bug?**

No.

**Comment by Manu1400** [comment]
Reproduced issue 👍

`  2:21  error  'Uint8Array' is not defined                     no-undef `

with js code:

```
var bufferLength = 10
var dataArray = new Uint8Array(bufferLength);
```
on config:
- node v10.15.1
- eslint v5.14.1
- npm 6.4.1

**Comment by not-an-aardvark** [comment]
Hi, thanks for the report. In order to enable ES2015 globals such as `Uint8Array`, you should add this to your config:

```diff
 extends: 'eslint:recommended'
 env:
   browser: true
+  es6: true

**Comment by AlexWayfer** [comment]
Oh, my bad. Sorry. I overlooked the standard.
